chapters/data-collection.tex

```diff
@@ -13,109 +13,337 @@
 \end{fullwidth}
 
 %------------------------------------------------
-\section{Survey development workflow}
-A well-designed questionnaire results from careful planning, consideration of analysis and indicators, close review of existing questionnaires, survey pilots, and research team and stakeholder review. There are many excellent resources on questionnaire design, such as from the World Bank's Living Standards Measurement Survey.
-\sidenote{Grosh, Margaret; Glewwe, Paul. 2000. Designing Household Survey Questionnaires for Developing Countries : Lessons from 15 Years of the Living Standards Measurement Study, Volume 3. Washington, DC: World Bank. © World Bank.\url{https://openknowledge.worldbank.org/handle/10986/15195 License: CC BY 3.0 IGO.}}
-The focus of this chapter is the particular design challenges for electronic surveys (often referred to as Computer Assisted Personal Interviews (CAPI)).
-\sidenote{\url{https://dimewiki.worldbank.org/wiki/Computer-Assisted\_Personal\_Interviews\_(CAPI)}}
+\section{Collecting primary data with development partners}
```

Comment:
Can we meet and agree on what we want to say in this section? This is the only section of this chapter (maybe the whole book) with no sources or sidenotes, and it feels like it was started to be written before we had an idea of what our recommendation. I think we should leave this as it is now, resolve all the conflicts this re-organization causes, and then meet and decide what we are recommending, and then and only then worry about the text. How does that sound?